src/hssm/distribution_utils/jax.py

```diff
@@ -114,6 +115,10 @@ def make_jax_logp_ops(
         The Jax function that calculates the VJP of the logp function.
     logp_nojit
         The non-jit version of logp.
+    n_params : optional
+        Only required when `extra_fields` are used, in which case the resulting Op will
+        not compute gradients with respect to the extra fields. Supply the number of
+        parameters that are used in the likelihood computation.
```

Comment:
Thanks for the context.
```suggestion
    n_params : optional
        Number of parameters used in the likelihood computation. Only required when `extra_fields` are used, in which case the resulting Op will not compute gradients with respect to the extra fields.
```